### PR TITLE
feat: Add optional user pin to PKCS #11 config.

### PR DESCRIPTION
**Pull request by clundin25**

### CODE REVIEW

File: internal/signer/linux/util/util.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ type PKCS11 struct {
 	Slot         string `json:"slot"`   // The hexadecimal representation of the uint36 slot ID. (ex:0x1739427)
 	Label        string `json:"label"`  // The token label (ex: gecc)
 	PKCS11Module string `json:"module"` // The path to the pkcs11 module (shared lib)
+	UserPin string `json:"user_pin"` // The path to the pkcs11 module (shared lib)
```
Comment (andyrzhao) [inline internal/signer/linux/util/util.go:37]: update comment

File: internal/signer/linux/util/util_test.go [hunk 2]
```diff
@@ -21,6 +21,11 @@ func TestLoadConfig(t *testing.T) {
 	if config.CertConfigs.PKCS11.PKCS11Module != want {
 		t.Errorf("Expected pkcs11_module is %v, got: %v", want, config.CertConfigs.PKCS11.PKCS11Module)
 	}
+
```
Comment (andyrzhao) [inline internal/signer/linux/util/util_test.go:24]: nit: remove extra blank line

**Review by andyrzhao — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks good overall!

**Comment by clundin25** [comment]
Thanks for the review!

**Review by andyrzhao — APPROVED** [review]
LGTM

**Pull request merged**